src/Application/Commands/ArchiveShoppingList/Validation/ArchiveShoppingListValidator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Lindyhopchris\ShoppingList\Application\Commands\ArchiveShoppingList\Validation;
+
+
```

Comment:
Formatting: there should only be one blank line here between the namespace and the import statements.